packages/veritone-react-common/src/components/LogoDetectionEngineOutput/story.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import styles from './story.styles.scss';
+import LogoDetectionEngineOutput from './';
+
+
+storiesOf('LogoDetectionEngineOutput', module).add('Base', () => {
+	return (
+		<div className={styles.container}>
+			<div className={styles.left}>
+				Left Box
+			</div>
+			<LogoDetectionEngineOutput 
+				className={styles.outputViewRoot}
+				data={mockData}
+			/>
+			<div className={styles.right}>
+				Right Box
+			</div>
+			<div className={styles.bottom}>
+				Bottom Box
+			</div>
+		</div>
+	);
+});
+
+const mockData = [
+	{
+		startTimeMs: 0,
+		endTimeMs: 9000,
+		sourceEngineId: '2dc5166f-c0ad-4d84-8a85-515c42b5d357',
+		sourceEngineName: 'Server-R',
+		taskId: 'e1fa7d7c-6f1c-480e-b181-68940509f070-fef496da-f36e-49ec-a304-426d96017ddf',
```

Comment:
I don't think we can assume this chunk of data is all associated to a single task.  In general, we don't want MDP or engine output viewers to know about either assets or tasks, just take time series data.  @AlexOVeritone any insight here?  I'll review the GraphQL branch in a bit to see what the connection is.